src/modules/formBuilder/components/FormEditorItemProperties.tsx

```diff
@@ -0,0 +1,182 @@
+import { Stack, Typography } from '@mui/material';
+import LabeledCheckbox from '@/components/elements/input/LabeledCheckbox';
+import TextInput from '@/components/elements/input/TextInput';
+import FormSelect from '@/modules/form/components/FormSelect';
+import { isPickListOption } from '@/modules/form/types';
+import {
+  localResolvePickList,
+  MAX_INPUT_AND_LABEL_WIDTH,
+} from '@/modules/form/util/formUtil';
+import { FormItem } from '@/types/gqlTypes';
+
+const dataCollectedAboutPickList =
+  localResolvePickList('DataCollectedAbout') || [];
+const disabledDisplayPickList = localResolvePickList('DisabledDisplay') || [];
+const inputSizePickList = localResolvePickList('InputSize') || [];
+
+interface FormEditorItemPropertiesProps {
+  item: FormItem;
+  onChangeProperty: (attr: keyof FormItem, newProperty: any) => void;
+}
+
+const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
+  item,
+  onChangeProperty,
+}) => {
+  return (
+    <>
+      <Typography>Properties</Typography>
+      <Stack
+        gap={2}
+        component='form'
+        onSubmit={() => {
+          console.log('submitted'); // TODO(#6103)
+        }}
+        sx={{ maxWidth: MAX_INPUT_AND_LABEL_WIDTH }}
+      >
+        <TextInput
+          label='Link ID'
+          value={item.linkId}
+          onChange={(e) => {
+            onChangeProperty('linkId', e.target.value);
+          }}
+          helperText='Unique ID for this form item'
```

Comment:
This should be validated on `^[a-zA-Z_$][a-zA-Z0-9_$]*$` (once https://github.com/greenriver/hmis-warehouse/pull/4129 goes in), plus on uniqueness across the form. Probably something to note for the future, or we could add a regex check now, your call.